liquidctl/driver/hydro_platinum.py

```diff
@@ -135,6 +135,8 @@ class HydroPlatinum(UsbHidDriver):
             {'fan_count': 2, 'fan_leds': 0}),
         (0x1b1c, 0x0c37, 'Corsair iCUE H150i Elite RGB',
             {'fan_count': 3, 'fan_leds': 0}),
+        (0x1b1c, 0x0c41, 'Corsair iCUE H150i Elite RGB (White)',
```

Comment:
From your PR description, it seems that you don't think differentiating between the two variants is warranted, and I tend I agree. We could keep both descriptions identical, and only leave a comment that the second PID is for the white variant. But both approaches are valid, so let me know which you, as someone that owns the device, prefers.

(If you decide to change the description, please remember to regenerate the udev rules once again).

Reply:
I think it's fine to keep "white" in the description, just so that if any user reports, the description will save some digging to find this device, and thus might have a slight chance for making it easier to debug.

Reply:
Fair enough. Merging, thanks!